README.md

```diff
@@ -392,6 +393,15 @@ Allows you to toggle between the normal GitHub contribution chart and an isometr
 
 A Chrome extension for infinite scrolling on GitHub's commit pages.
 
+## [Like On Github](https://github.com/Idnan/like-on-github)
+<a href="https://chrome.google.com/webstore/detail/lovely-forks/ialbpcipalajnakfondkflpkagbkdoib"><img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" width="24" /></a>
+
+Chrome extension that lets you save links to a Github repository
```

Comment:
Can you explain it a little be more in depth what the extension does. That would be helpful. 

Btw love this idea, well done 👍